whatsapp_theming/README.md

```diff
@@ -0,0 +1,17 @@
+# whatsapp_theming
+An example app which demonstrates dark light and auto switching of theme options just like what whatsapp provides
+
+## Getting Started
+
+I have used shared preferences and theme provider package to develop such kind of dynamic theme changing hope it helps!
+
+![video](demovideo.mp4)
```

Comment:
I can't comment on the video file itself, but `demovideo.mp4` is too big. It should be compressed further. Use ffmpeg to compress it further maintaining aspect ratio and height set to 320px

Reply:
We are using gifs everywhere. Don't bundle video files.

Use https://ezgif.com/video-to-gif to convert video to gif. Make sure the size is set to 320 height with aspect ratio width.